Pipfile.lock

```diff
@@ -51,16 +72,6 @@
             "index": "pypi",
             "version": "==2.3.2"
         },
-        "pyqt5": {
```

Comment:
thought: instead of removing `pyqt5` entirely we should probably move `pyqt5` to the development requirements of the Pipfile, since we still want to make sure it's installed in dev/ci/test but not prod/build

Reply:
This still can create false positive as dev/ci will be different that production. We should use the same system provided package everywhere.

Reply:
Using a pipfile dev dependency enables e.g. macOS developers. If the _only_ way to run the dev env is to use a Debian-based OS, that's going to be a problem. Probably worth accepting a small bit of divergence in order to accommodate dev workflows here.

Reply:
yeah, i mean, i'm open to other suggestions here but the only other real option (other than use debian, ha) is to instruct developers on macOS to install qt5 separately in the dev env

Reply:
dumb question here -- can we use docker containers and VNC ? if we use system level PyQt versions arent we going to have mismatch between different developers? or is pyqt so rock stable that this shouldn't be a problem? (i swear im not trolling here!)

Reply:
Good point @msheiny, with system packages, it's not just "Debian-based OS" that'd be the requirement, but _specifically Debian Stretch._ Unless you're running Qubes, that's a lot to ask from contributors looking to write a bit of Python code.

The container/VNC strategy is interesting, but given the outstanding deadlines we're working with—particularly the upcoming audit—I suggest we preserve the pyqt dependency in the pipfile via dev dependencies as @redshiftzero suggests.

Reply:
I just pushed a commit which adds PyQt5 to the dev dependencies.

Reply:
+1 on all this...WRT PyQT stability issues... I'd say it's "mostly" rock solid, except when it isn't. You can draw your own conclusions. ;-)